var/spack/repos/builtin/packages/mochi-thallium/package.py

```diff
@@ -67,6 +78,7 @@ class MochiThallium(CMakePackage):
         depends_on("cereal@1.3.1:", when="@0.10.1:")
     # thallium relies on std::decay_t
     conflicts("%gcc@:4.9.0")
+    conflicts("~cereal", when="@0.14.0:", msg="Thallium 0.14.0 and above requires Cereal")
```

Comment:
(Suggestion) Move up to immediately after the variant for context.

Reply:
Thanks, I moved it.

Reply:
Regarding the `depends_on("mochi-margo@...")`, I forgot to add one actually. This package should reflect the one we keep in our mochi-spack-packages repository: https://github.com/mochi-hpc/mochi-spack-packages/blob/main/packages/mochi-thallium/package.py